logstash-core/spec/logstash/agent_spec.rb

```diff
@@ -296,6 +296,24 @@
     end
   end
 
+  context "--config-test" do
+    let(:cli_args) { ["-t", "-e", pipeline_string] }
+
+    context "with a good configuration" do
+      let(:pipeline_string) { "input { } filter { } output { }" }
```

Comment:
I see this config string being used in many places, I think I count at less 4 in this same spec. Can we dry it out a bit?